### PR TITLE
Add OKR

### PR DESCRIPTION
**Pull request by domenicosolazzo**
A curated list about the OKR methodology (Objective-Key Results).

### CODE REVIEW

**Comment by aleksandar-todorovic** [comment]
Huh, where do I even start? MIT is not a license suitable for these kinds of things, no contributing guidelines, no description about why something is awesome, no info about what OKR means in the list itself. 

Please, take your time to read the awesome manifesto and a guide for creating lists (you'll find them in this repository). Once your list complies to them, feel free to ping us here.

**Comment by domenicosolazzo** [comment]
Hi @aleksandar-todorovic , sorry my delayed answer.

Thank you very much for your feedback. I went through your suggestions and I modified the list accordingly.

Would you mind to take a look and check if it is ok now? Let me know if there is something more for making this list more awesome :)

**Comment by aleksandar-todorovic** [comment]
Looks a lot better now! I'll wait for someone else to confirm this, but I think that this list should be accepted because it is indeed awesome.

File: readme.md [hunk 1]
```diff
@@ -370,6 +370,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [MQTT](https://github.com/hobbyquaker/awesome-mqtt) *(an "Internet of Things" connectivity protocol)*
 - [Hacking Spots](https://github.com/diasdavid/awesome-hacking-spots)
 - [For Girls](https://github.com/cristianoliveira/awesome4girls)
+- [OKR](https://github.com/domenicosolazzo/awesome-okr)
```
Comment (sindresorhus) [inline readme.md]: Can you add a short and succinct description? See above for example on how it's done.

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

File: readme.md (hunk 1 above)
Comment (sindresorhus) [inline readme.md]: Title could also be `OKR methodology` I think.

**Comment by domenicosolazzo** [comment]
@sindresorhus: made the changes. What do you think?

**Pull request closed without merging**